### ISSUE TITLE
Export geojson fails on inf values

### ISSUE THREAD

**Issue by BramSchnitzler** [openearth/flowmap#55]
* Flowmap generator version:1.0.0
* Python version:3.5
* Operating System: Windows 

### Description

Tried to export geojson results taking a valid_range into account. Exporting fails on infinite values. These infinite values are caused during exporting tables taking a valid_range into account. 

if valid_range = None, then exporting a geojson goes without a problem.

### What I Did

```
import flowmap.formats.ugrid
ugrid_dem = flowmap.formats.ugrid.UGrid(path = r'd:\Project\P940\05_Analysis\Resultaten\1131\Leerdam\leerdamwest_map.nc', dem = r'd:\Project\P940\05_Analysis\GIS\AHN\07_resultaat\C24\1131\resultaten\aw_ahn_def.tif')

t = 5
method = 'waterlevel'
format = '.geojson'


"""compute refined waterlevel using detailled dem, using subgrid or interpolate method"""
dem = read_dem(ugrid_dem.options['dem'])
grid = ugrid_dem.ugrid
data = ugrid_dem.waterlevel(t)

table_name = ugrid_dem.generate_name(
    ugrid_dem.path,
    suffix='.nc',
    topic='tables'
)
table_path = pathlib.Path(table_name)

ugrid_dem.export('id_grid')
ugrid_dem.export('tables', valid_range = [-6,10]) 

tables = subgrid.import_tables(str(table_path))

feature_collection = subgrid.compute_features(grid, dem, tables, data, method=method)
new_name = ugrid_dem.generate_name(
    ugrid_dem.path,
    suffix=format,
    topic=method,
    counter=t
)


crs = geojson.crs.Named(
    properties={
        "name": "urn:ogc:def:crs:EPSG::{:d}".format(ugrid_dem.src_epsg)
    }
)
feature_collection['crs'] = crs
with open(new_name, 'w') as f:
    geojson.dump(feature_collection, f, cls=flowmap.formats.ugrid.CustomEncoder, allow_nan=False)

Traceback (most recent call last):

  File "<ipython-input-84-a9049c0c69e5>", line 8, in <module>
    geojson.dump(feature_collection, f, cls=flowmap.formats.ugrid.CustomEncoder, allow_nan=False)#, ignore_nan=True)#,default=str)

  File "C:\Users\schnitzler\AppData\Local\Continuum\anaconda2_64\envs\P940_flowmap_05feb\lib\site-packages\geojson\codec.py", line 27, in dump
    fp, cls=cls, allow_nan=allow_nan, **kwargs)

  File "C:\Users\schnitzler\AppData\Local\Continuum\anaconda2_64\envs\P940_flowmap_05feb\lib\json\__init__.py", line 179, in dump
    for chunk in iterable:

  File "C:\Users\schnitzler\AppData\Local\Continuum\anaconda2_64\envs\P940_flowmap_05feb\lib\json\encoder.py", line 430, in _iterencode
    yield from _iterencode_dict(o, _current_indent_level)

  File "C:\Users\schnitzler\AppData\Local\Continuum\anaconda2_64\envs\P940_flowmap_05feb\lib\json\encoder.py", line 404, in _iterencode_dict
    yield from chunks

  File "C:\Users\schnitzler\AppData\Local\Continuum\anaconda2_64\envs\P940_flowmap_05feb\lib\json\encoder.py", line 325, in _iterencode_list
    yield from chunks

  File "C:\Users\schnitzler\AppData\Local\Continuum\anaconda2_64\envs\P940_flowmap_05feb\lib\json\encoder.py", line 404, in _iterencode_dict
    yield from chunks

  File "C:\Users\schnitzler\AppData\Local\Continuum\anaconda2_64\envs\P940_flowmap_05feb\lib\json\encoder.py", line 396, in _iterencode_dict
    yield _floatstr(value)

  File "C:\Users\schnitzler\AppData\Local\Continuum\anaconda2_64\envs\P940_flowmap_05feb\lib\json\encoder.py", line 241, in floatstr
    repr(o))

ValueError: Out of range float values are not JSON compliant: inf

```

**Comment by SiggyF** [comment]
Thanks I will test this.

**Comment by SiggyF** [comment]
This is caused by disabling `, ignore_nan=True)`. If there are nan/inf values returned they should be mapped to null. These values can be caused by a face_area of 0, for cells that do not have valid dem pixels.

**Comment by BramSchnitzler** [comment]
I removed ``` , ignore_nan=True)``` since the json version installed for my Python does not support ``` , ignore_nan=True)``` (Json 2.0.9). Replace geojson with simplejson did the trick for me. Requires installation and importation of simplejson though



```
with open(waterlevel_name, 'w') as f:
    geojson.dump(feature_collection, f, cls=CustomEncoder, allow_nan=False, ignore_nan=True)


Traceback (most recent call last):

  File "<ipython-input-59-26245ea970c5>", line 2, in <module>
    geojson.dump(feature_collection, f, cls=CustomEncoder, allow_nan=False, ignore_nan=True)

  File "C:\Users\schnitzler\AppData\Local\Continuum\anaconda2_64\envs\P940_flowmap_05feb\lib\site-packages\geojson\codec.py", line 27, in dump
    fp, cls=cls, allow_nan=allow_nan, **kwargs)

  File "C:\Users\schnitzler\AppData\Local\Continuum\anaconda2_64\envs\P940_flowmap_05feb\lib\json\__init__.py", line 176, in dump
    default=default, sort_keys=sort_keys, **kw).iterencode(obj)

TypeError: __init__() got an unexpected keyword argument 'ignore_nan'


with open(waterlevel_name, 'w') as f:
    simplejson.dump(feature_collection, f, cls=CustomEncoder, allow_nan=False, ignore_nan=True)



```

**Comment by SiggyF** [comment]
Geojson uses simplejson if [available](https://github.com/frewsxcv/python-geojson/blob/master/geojson/codec.py#L1). I will add this to the list of dependencies. Then it should work with the code as is.

**Comment by SiggyF** [comment]
Fixed in v1.0.2
